javascript/src/main/pages/Courses/EditCourse.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import useSWR from "swr";
+import CourseForm from "main/components/Courses/CourseForm";
+import { fetchWithToken } from "main/utils/fetch";
+import { useAuth0 } from "@auth0/auth0-react";
+import Loading from "main/components/Loading/Loading";
+import { useParams, useHistory } from "react-router-dom";
+import { buildUpdateCourse } from "main/services/Courses/CourseService";
+import { cache } from "swr"
```

Comment:
This import can be combined with the default import for `swr` above for something like:

```jsx
import useSWR, { cache } from "swr"
```